package.json

```diff
@@ -6,10 +6,13 @@
   "scripts": {
     "test": "jest",
     "build": "tsc",
-    "build:clean": "rm -rf dist/ && npm run build",
+    "build:clean": "rimraf dist/ && npm run build",
```

Comment:
I wonder why we prefer `rimraf`?

Reply:
We use it on @zeplin/cli, I wanted to have consistency among all CLI packages. On a second thought, I put it on @zeplin/cli if I were to develop on Windows, considering I already use WSL while developing, there is no reason to use rimraf.

Reply:
Oh, I see we need `rimraf` for Windows. I didn't know that. Thanks!